### ISSUE TITLE
Add `verbose` argument to `xdem.spstats.patches_method` and/or change prints to `tqdm`

### ISSUE THREAD

**Issue by erikmannerfelt** [GlacioHack/xdem#132]
Whenever tests fail, we get this beautiful output:
```
Datasets exist
Number of valid pixels: 1138155
Found valid cadrant 1 (maximum: 1000)
Found valid cadrant 2 (maximum: 1000)
Found valid cadrant 3 (maximum: 1000)
Found valid cadrant 4 (maximum: 1000)
Found valid cadrant 5 (maximum: 1000)
Found valid cadrant 6 (maximum: 1000)
Found valid cadrant 7 (maximum: 1000)
Found valid cadrant 8 (maximum: 1000)
Found valid cadrant 9 (maximum: 1000)
Found valid cadrant 10 (maximum: 1000)
Found valid cadrant 11 (maximum: 1000)
Found valid cadrant 12 (maximum: 1000)
Found valid cadrant 13 (maximum: 1000)
Found valid cadrant 14 (maximum: 1000)
Found valid cadrant 15 (maximum: 1000)
Found valid cadrant 16 (maximum: 1000)
Found valid cadrant 17 (maximum: 1000)
Found valid cadrant 18 (maximum: 1000)
Found valid cadrant 19 (maximum: 1000)
Found valid cadrant 20 (maximum: 1000)
Found valid cadrant 21 (maximum: 1000)
Found valid cadrant 22 (maximum: 1000)
Found valid cadrant 23 (maximum: 1000)
Found valid cadrant 24 (maximum: 1000)
Found valid cadrant 25 (maximum: 1000)
Found valid cadrant 26 (maximum: 1000)
Found valid cadrant 27 (maximum: 1000)
Found valid cadrant 28 (maximum: 1000)
Found valid cadrant 29 (maximum: 1000)
Found valid cadrant 30 (maximum: 1000)
Found valid cadrant 31 (maximum: 1000)
Found valid cadrant 32 (maximum: 1000)
Found valid cadrant 33 (maximum: 1000)
Found valid cadrant 34 (maximum: 1000)
Found valid cadrant 35 (maximum: 1000)
Found valid cadrant 36 (maximum: 1000)
Found valid cadrant 37 (maximum: 1000)
Found valid cadrant 38 (maximum: 1000)
Found valid cadrant 39 (maximum: 1000)
Found valid cadrant 40 (maximum: 1000)
Found valid cadrant 41 (maximum: 1000)
Found valid cadrant 42 (maximum: 1000)
Found valid cadrant 43 (maximum: 1000)
Found valid cadrant 44 (maximum: 1000)
Found valid cadrant 45 (maximum: 1000)
Found valid cadrant 46 (maximum: 1000)
Found valid cadrant 47 (maximum: 1000)
```
etc, with not 47, but 1000 lines!

It would be great both for users and for developers who look at the logs to hide these 1000 print calls!

I suggest that either a `verbose` setting is added which should be `False` in the test and/or the print statements are replaced with a `tqdm` progress bar.

Basically, I would be happy with any solution that removes or at least shortens these outputs from the tests!!

**Comment by rhugonnet** [comment]
What a beautiful output, I wonder who coded this... :laughing:
Adding to my to do list!

**Comment by rhugonnet** [comment]
Closed by #136
